Application/EdFi.Ods.Api.IntegrationTestHarness/Startup.cs

```diff
@@ -13,7 +16,8 @@ public class Startup : OdsStartupBase
         public Startup(IWebHostEnvironment env, IConfiguration configuration)
             : base(env, configuration)
         {
-
+            AssemblyLoader.EnsureLoaded<Marker_EdFi_Ods_Composites_Test>();
+            AssemblyLoader.EnsureLoaded<Marker_EdFi_Ods_Profiles_Test>();
```

Comment:
This change is not needed, we load all assemblies in the folder automatically now
```suggestion
```